### PR TITLE
srb-rbi with no args just prints usage (#1006)

### PR DESCRIPTION
**Pull request by pje**
Implements #1006.

AFAICT there were no tests for `srb-rbi`! So I didn't write any.

### CODE REVIEW

**Comment by jez** [comment]
> AFAICT there were no tests for `srb-rbi`! So I didn't write any.

Yeah there are regrettably no tests for any subcommand other than `srb init` :(

**Comment by DarkDimius** [comment]
I'll refer to @jez and @ptarjan as they worked on UX for `srb`.
I remember discussions that `srb` by default in setup project might be equivalent to `srb tc` but I'm not sure where did they land.

**Comment by pje** [comment]
> Yeah there are regrettably no tests for any subcommand other than srb init :(

I'd be happy to add like a simple minitest unit test for this! I see that you're using minitest elsewhere.

**Comment by jez** [comment]
> I'd be happy to add like a simple minitest unit test for this! I see that you're using minitest elsewhere.

We don't want to use minitest for this. We want to test the CLI itself. I'd prefer to leave this untested, unless you build something like the existing `gems/sorbet/test/snapshot/driver.sh` test harness.

See [the README](https://github.com/sorbet/sorbet/blob/master/gems/sorbet/README.md#writing-tests) to see what i mean.

**Comment by pje** [comment]
@jez: I've rebased from master and reduced the diff size (most of that was just due to re-ordering some method definitions). LMKWYT!

File: gems/sorbet/bin/srb-rbi [hunk 1]
```diff
@@ -73,7 +102,7 @@ actions:
       STDOUT.write(emojify("❔", "Would you like to continue? [Y/n] "))
       if STDIN.isatty && STDOUT.isatty
         begin
-          input = Kernel.gets&.strip
+          input = STDIN.gets&.strip
```
Comment (pje) [inline gems/sorbet/bin/srb-rbi]: Had to change because of [this](https://stackoverflow.com/a/10523615)

Comment (adsteel) [reply gems/sorbet/bin/srb-rbi]: Just ran into this issue myself running `srb rbi update`. If this PR gets delayed I'd love to see this change as a separate PR.

Comment (adsteel) [reply gems/sorbet/bin/srb-rbi]: I now see there's another PR addressing this as well. https://github.com/sorbet/sorbet/pull/1066

**Comment by pje** [comment]
@jez: Anything blocking this getting merged?

**Comment by jez** [comment]
@pje sorry for the delay. this looks great to me, thanks!

i'll merge when the test pass.

**Comment by jez** [comment]
@pje I had to refactor some things to get the tests to pass. You can see the diff in https://github.com/sorbet/sorbet/pull/1063/commits/93bb6280d0fbbf73ca21ae44a95bc936d3d6f94a . The tl;dr is that because the `parse_command` method didn't exit the program, it would cause the usage output to show up even after a 100% successful run of `srb init`. Our tests failed because of this, and it should be fixed now. I'm just waiting for the tests to pass to be sure.

**Comment by pje** [comment]
Thanks for the assist, @jez! 🙇

**Pull request merged**